data/transactions/logic/README.md

```diff
@@ -275,6 +277,7 @@ Some of these have immediate data in the byte or bytes after the opcode.
 | 53 | GlobalNumByteSlice | uint64 | Number of global state byteslices in ApplicationCall. LogicSigVersion >= 3. |
 | 54 | LocalNumUint | uint64 | Number of local state integers in ApplicationCall. LogicSigVersion >= 3. |
 | 55 | LocalNumByteSlice | uint64 | Number of local state byteslices in ApplicationCall. LogicSigVersion >= 3. |
+| 56 | AppProgramExtraPages | uint64 |  |
```

Comment:
Missing a description for AppProgramExtraPages?

Reply:
Yes, I think that's the case as this was generated. Is there some description you'd like me to add and include in this PR?

Reply:
maybe Number of extra pages in ApplicationCall. LogicSigVersion >= 4

Reply:
Will do!

Reply:
Yes, ideally the doc will say something along the lines of "Number of additional pages for each of the application's approval and clear state programs. A page is 1024 bytes." Does that imply strongly enough that 1 extra page means 2048 more bytes total?

Reply:
I'm not sure it does – how about
> Number of additional pages for each of the application's approval and clear state programs. An ExtraProgramPages of 1 means 2048 more total bytes, or 1024 for each program.

It is very long though, so hopefully we can pare it down.